src/types/types.ts

```diff
@@ -1584,6 +1584,48 @@ export interface SendPrivateTransactionOptions {
   fast: boolean;
 }
 
+export enum AssetType {
+  NATIVE = 'NATIVE',
+  ERC20 = 'ERC20',
+  ERC721 = 'ERC721',
+  ERC1155 = 'ERC1155',
+  SPECIAL_NFT = 'SPECIAL_NFT'
+}
+
+export enum ChangeType {
+  APPROVE = 'APPROVE',
+  TRANSFER = 'TRANSFER'
+}
+
+interface AssetChange {
```

Comment:
rename AssetChanges to `RawSimulateAssetChangesResposne` and move it to `raw-interfaces.ts`.